tests/pgsmo_tests/test_server_connection.py

```diff
@@ -4,34 +4,38 @@
 # --------------------------------------------------------------------------------------------
 
 import unittest
+import unittest.mock as mock
 
-import pgsmo.utils as pgsmo_utils
 import tests.pgsmo_tests.utils as utils
+from tests.utils import MockConnection as MockPsycopgConnection
+from ossdbtoolsservice.driver.types.psycopg_driver import PostgreSQLConnection
 
 
 class TestServerConnection(unittest.TestCase):
     def test_server_conn_init(self):
         # Setup: Create a mock connection with an 'interesting' version
-        mock_conn = utils.MockConnection(None, version='100216')
+        mock_conn = MockPsycopgConnection()
+        mock_conn.server_version = '100216'
 
         # If: I initialize a server connection
         # noinspection PyTypeChecker
-        server_conn = pgsmo_utils.querying.ServerConnection(mock_conn)
+        with mock.patch('psycopg2.connect', new=mock.Mock(return_value=mock_conn)):
+            server_conn = PostgreSQLConnection({})
 
         # Then: The properties should be properly set
         self.assertEqual(server_conn._conn, mock_conn)
         self.assertEqual(server_conn.connection, mock_conn)
-        expected_dict = {'dbname': 'postgres', 'host': 'localhost', 'port': '25565', 'user': 'postgres'}
-        self.assertDictEqual(server_conn._dsn_parameters, expected_dict)
```

Comment:
Before we were asserting the dsn_parameters which has dbname, host, post and user. We should assert all of them separately.